lib/friendly_shipping/services/ups/serialize_address_snippet.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class Ups
+      class SerializeAddressSnippet
+        class <<self
```

Comment:
missing space

```suggestion
        class << self
```

Also I recognized that sometimes you use `def self.call` sometimes `class << self`, even for "classes" with single a method.